Formula/systemc.rb

```diff
@@ -1,15 +1,10 @@
 class Systemc < Formula
   desc "Core SystemC language and examples"
   homepage "https://accellera.org/"
-  url "https://www.accellera.org/images/downloads/standards/systemc/systemc-2.3.2.tar.gz"
```

Comment:
```
$ curl -I 'https://www.accellera.org/images/downloads/standards/systemc/systemc-2.3.4.tar.gz'
HTTP/1.1 302 Found
Date: Sat, 18 Feb 2023 22:45:57 GMT
Server: Apache
Set-Cookie: 06fc5174bd54493c1f32783c2fd16f01=e793f8dd598cb1cbe20846ab88665845; path=/; HttpOnly
Location: /404
```

Reply:
Also the 2.3.4 download link on `https://www.accellera.org/downloads/standards/systemc` page already points to the github archive file.